src/main.jsx

```diff
@@ -1,12 +1,10 @@
-import { StrictMode } from "react";
+// import { StrictMode } from "react";
```

Comment:
We probably shouldn't disable this.  You could implement this wrapper if you don't want to deal with the double calls on mount: https://taig.medium.com/prevent-react-from-triggering-useeffect-twice-307a475714d7

Reply:
We will eventually swap to rtk-query which will eliminate this as a problem

Reply:
reapplied StrickMode